docs/ref-manual/cvl/types.md

```diff
@@ -89,6 +92,94 @@ multi-dimensional arrays
 reference types
 ```
 
+Types Defined in Solidity Contracts
+-----------------------------------
+
+Solidity supports several user defined types:
+* Enums
+* User Defined Value Types (type aliases)
+* Structs
+
+These types can be accessed by referencing the contract where they are defined. To reference a contract remember to include `using ContractName as myLocalContractName`. The file containing the contract `ContractName` will have to be included in your build.
+
+User defined types may then be referenced from that contract: `myLocalContractName.myTypeName`. For types declared at the file level, any contract which imports that file may be used. Note that new user defined types may *not* be declared within a specification.
+
+### Enum Types
+Consider the following enum declaration and spec preamble:
+
+```
```

Comment:
specify `cvl` or `solidity` after the triple backtick.  See `README.md` in the top-level directory for an example